lib/everypolitician.rb

```diff
@@ -108,25 +108,22 @@ def popolo
 
     def legislative_periods
       @legislative_periods ||= raw_data[:legislative_periods].map do |lp|
-        LegislativePeriod.new(lp, self, self.country)
+        LegislativePeriod.new(lp, self, country)
       end
     end
 
     def directory
-      @directory = raw_data[:sources_directory].split('/')[1,2].join('/')
+      @directory = raw_data[:sources_directory].split('/')[1, 2].join('/')
     end
 
     def self.from_sources_dir(dir)
       @index_by_sources ||= EveryPolitician.countries.map(&:legislatures).flatten.group_by(&:directory)
       @index_by_sources[dir][0]
     end
 
-    private
-
-    def to_UTC_time(string_time)
-      Time.at(string_time.to_i).gmtime
+    def lastmod
+      Time.at(lastmod_str.to_i).gmtime
     end
```

Comment:
I really like the philosophy behind this change! :+1: 
However not sure about calling the field `lastmod_str`, as it reminds me of the [hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation) :laughing: 
Naming is hard!